pkgs/by-name/pi/pixivfe/package.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitea,
+  makeBinaryWrapper,
+}:
+
+# NOTE: Please do not forget to manually update these fields!
+let
+  revision = "19b09e7";
+  date = "2025.01.10";
+in
+buildGoModule rec {
+  pname = "pixivfe";
+  version = "2.11";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "VnPower";
+    repo = "PixivFE";
+    rev = "v${version}";
+    hash = "sha256-i5w+LagVrBsG3d6MMIyG8rgeyd3hlPkLhhW6O3CwF7g=";
+  };
+
+  vendorHash = "sha256-5TmvRzdDAxvsWPBJMXYg0hCleMK1sT0aGg5rdSOMUbU=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X codeberg.org/vnpower/pixivfe/v2/config.revision=${date}-${revision}"
```

```suggestion
    "-X codeberg.org/vnpower/pixivfe/v2/config.revision=1070.01.01-00000000"
```
What speaks against this?